package.json

```diff
@@ -77,18 +82,21 @@
     "grunt-protractor-webdriver": "^0.2.5",
     "grunt-sass": "3.1.0",
     "jasmine-core": "^3.5.0",
-    "karma": "^4",
+    "karma": "^6.3.9",
     "karma-chrome-launcher": "^3.1.0",
     "karma-cli": "^2.0.0",
     "karma-jasmine": "^4.0.1",
-    "karma-typescript": "4.1.1",
-    "protractor": "^7.0.0",
+    "karma-typescript": "^5.5.2",
+    "protractor": "^3.3.0",
     "puppeteer": "^5.4.1",
     "sass": "^1.42.1",
     "serve-static": "^1.14.1",
-    "ts-loader": "^8.0.7",
+    "ts-loader": "^8.1.0",
     "typescript": "^3.7",
-    "webpack": "^5.3.2",
+    "webpack": "^5.31.2",
     "webpack-cli": "^4.6.0"
+  },
+  "dependencies": {
+    "yarn": "^1.22.17"
```

Comment:
should be a devDependencies, not dependency (that user need to install)

also yarn.lock should be updated instead... not sure why you had (?) to change the rev of the libs...